js/build.js

```diff
@@ -0,0 +1,344 @@
+        // we have it for now only in record container
+        return Fliplet.DataSources.connect(dataSourceId).then(function (connection) {
+          return connection.findById(dataSourceEntryId).then(function (record) {
+            if (!checkArrayLinks(record.data[columnName])) {
+              $(document).find('[data-helper="file-list"]').html(`<p>There are no ${type}s</p>`)
+            } else {
+              var fileIDs = record.data[columnName].map(function (file) {
+                var url = typeof file === 'string'
+                  ? file
+                  : file.url;
+
+                return Fliplet.Media.getIdFromRemoteUrl(url);
+              });
+
+              // TODO check how to escape this call
+              return Fliplet.Media.Files.getAll({
+                files: fileIDs,
+                fields: ['name', 'url', 'metadata', 'createdAt']
+              }).then(function (files) {
+                var filesInfo = files.map(function (file) {
+                  const extension = file.name.split('.').pop().toLowerCase();
+                  var type = '';
+                  if (extension === 'jpg' || extension === 'jpeg' || extension === 'png' || extension === 'gif') {
+                    type = 'image';
+                  } else if (extension === 'mp4' || extension === 'avi' || extension === 'mkv' || extension === 'mov') {
+                    type = 'video';
+                  } else {
+                    type = 'file';
+                  }
+                  return {
+                    type,
+                    name: file.name,
+                    size: file.metadata.size,
+                    uploaded: file.createdAt,
+                    url: file.url
+                  };
+                }).sort(sortFilesByName);
+                if (type == 'Image') {
+                  var data = {
+                    images: filesInfo.map(el => {
+                      return {
+                        // title: el.name,
+                        url: Fliplet.Media.authenticate(el.url)
+                      }
+                    }),
+                    options: {
+                      index: 0,
+                      errorMsg: 'The photo cannot be loaded'
+                    }
+                  };
+                  var images = data.images.map((el, index) => {
+                    return `<div class="image-item-container" data-index="${index}">
+                              <img src="${el.url}" />
+                            </div>`
+                  })
+                  $(document).find('[data-helper="file-list"]').append(`
+                   <div class="image-container">
+                   ${images.join('')}
+                   <div>
+                  `);
+
+                  $(document).find('.image-item-container').off('click');
+                  $(document).find('.image-item-container').on('click', function () {
```

Comment:
You should be able to chain `.off().on()`

Reply:
I thought it would be more legible in two lines. Done.